src/components/NotLoggedCabinet/NotLoggedCabinet.js

```diff
@@ -8,7 +8,8 @@ import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 
```

Comment:
Please, rename the file and folder to the `not-logged-cabinet` instead of `NotLoggedCabinet`

Reply:
Renamed